illumio/util/functions.py

```diff
@@ -178,63 +175,46 @@ def convert_protocol(protocol: str) -> int:
         raise IllumioException('Invalid protocol name: {}'.format(protocol))
 
 
-def validate_int(val: typing.Any, minimum: int=0, maximum: int=sys.maxsize) -> None:
+def validate_int(val: typing.Any, minimum: int = 0, maximum: int = sys.maxsize) -> None:
     """Validates a given value is an integer and is within min <= val <= max.
-
     Args:
         val (Any): value to validate.
-        min (int, optional): validation lower bound. Defaults to 0.
-        max (int, optional): validation upper bound. Defaults to sys.maxsize.
+        minimum (int, optional): validation lower bound. Defaults to 0.
+        maximum (int, optional): validation upper bound. Defaults to sys.maxsize.
 
     Raises:
         IllumioIntegerValidationException: if an invalid value is provided.
     """
     try:
-        valid = minimum <= int(val) <= maximum
-    except:  # catch the ValueError for invalid values
-        valid = False
-    if not valid:
+        if not minimum <= int(val) <= maximum:
```

Comment:
Why was this changed?

Reply:
seems like it could be optimized - is the optimized version not better?